terraform/files/bin/apply_cert_manager.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 . ~/bin/cccfg.inc
 export KUBECONFIG=~/.kube/config
+. ~/$CLUSTER_NAME/harbor-settings
 
 echo "Deploy cert-manager to $CLUSTER_NAME"
 # cert-manager
 DEPLOY_CERT_MANAGER=$(yq eval '.DEPLOY_CERT_MANAGER' $CCCFG)
+if test "$DEPLOY_CERT_MANAGER" = "false" -a "$DEPLOY_HARBOR" = "true" -a -n "$HARBOR_DOMAIN_NAME"; then
+  DEPLOY_CERT_MANAGER="true"
```

Comment:
Consider adding a log message that Harbor forced the cert-manager deployment. `echo ...`

Reply:
Addressed in 3b86ce283d77c80d1b6ee3415379ea1035ef7699